bindings/rascal/utils/misc.py

```diff
@@ -0,0 +1,17 @@
+"""Miscellaneous useful utilities"""
+
+
+def is_notebook():
```

Comment:
I don't really see why we have this function in librascal ... Is this used anywhere else in the code or is this just a convenience function for librascal users?

Reply:
It was used to decide whether to import `tqdm` in the command-line version or the notebook version, in one of the tutorials (yet to be merged, I think). I know we wanted to make the dependency on `tqdm` optional, but if it is present, we will probably want to import the appropriate version.

Reply:
It was one of the i-PI tutorials actually, see e.g.: https://github.com/cosmo-epfl/librascal/pull/307/commits/af2c7bd0b1e3917f8c7aaab10b850418f500d46a

Reply:
Ok, ideally I would move this function to the tutorial code itself, and not inside librascal. But this can happen as part of #307

Reply:
That PR is scheduled to be merged soon anyway (I split this fix off from that PR because I thought it was more urgent)